api/src/kots_app/kots_ffi.ts

```
@@ -325,25 +325,26 @@ async function saveUpdateVersion(archive: string, app: KotsApp, stores: Stores)
   }
 }
 
-export async function kotsAppFromLicenseData(licenseData: string, name: string, downstreamName: string, stores: Stores): Promise<KotsApp> {
+export async function kotsAppFromLicenseData(licenseData: string, name: string, downstreamName: string, stores: Stores): Promise<KotsApp | undefined> {
```

would make more sense to throw an error instead of returning undefined